helpers/helpers.go

```diff
@@ -0,0 +1,43 @@
+package helpers
```

Comment:
I'm generally against creating packages named "helpers" or "util" since they become dumping grounds for lots of unrelated things. Better to have lots of tiny package with specific names. Like these two functions could go in a `pki` package or a `certs` package.

Reply:
That's fair. I had a hard time thinking of what name to put this under. I was going to use `test/` originally but it felt wrong to be importing from a test package in non-test code.

`pki` would be fine I guess.